### ISSUE TITLE
[A11y Library Fix] - "eCR Library" should be a header

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/dibbs-ecr-viewer#288]
## What needs to be done

Change the "eCR Library" text to be an HTML header element (it's currently a span).

## Why it needs to be done

This is an accessibility issue - screenreader users would not "see" the eCR Library heading in their content scan.

<img width="876" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeWZUQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--807608b9818be7c55456fcb51596e713377800ee/Screenshot%202025-02-11%20at%2011.57.05%E2%80%AFAM.png" alt="Screenshot 2025-02-11 at 11.57.05 AM.png" />

See Axe scan issue: https://cdc-devtools.dequecloud.com/issues/972b0840-6fd7-4f34-a4fd-f44ba2aba632

## Acceptance Criteria

- [ ] "eCR Library" element is a `h2` element
